openpype/pipeline/create/context.py

```diff
@@ -861,6 +865,17 @@ def reset_plugins(self, discover_publish_plugins=True):
                     "Using first and skipping following"
                 ))
                 continue
+
+            # Filter by host name
+            if (
+                creator_class.host_name
+                and creator_class.host_name != self.host_name
+            ):
+                self.log.info((
+                    "Creator's host name is not supported for current host {}"
+                ).format(creator_class.host_name, self.host_name))
+                continue
```

Comment:
For consistency, shouldn't we support similar host name filtering like Pyblish does? Like allowing `hosts = ["maya", "houdini"]` and maybe have default value of `hosts = ["*"]`?

Reply:
I don't think you can have same creator plugins for 2 different hosts....

Reply:
Ah, of course that might make a lot less sense.